pkg/wasm/install/endpoint.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Layotto Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package install
+
+import (
+	"context"
+	"errors"
+
+	"mosn.io/layotto/pkg/wasm"
+
+	"mosn.io/pkg/log"
+
+	"mosn.io/layotto/pkg/filter/stream/common/http"
```

Comment:
also format this

Reply:
what do you mean，gofmt already executed, there is any special code style？